core/container/externalbuilder/copy_test.go

```diff
@@ -19,23 +20,35 @@ import (
 
 var _ = Describe("copy", func() {
 	var (
-		logger                             *flogging.FabricLogger
-		srcRootDir, srcSubDir, destRootDir string
-		srcRootFile, srcSubFile            *os.File
-		err                                error
+		logger                  *flogging.FabricLogger
+		srcRootDir, destRootDir string
+		err                     error
```

Comment:
Oh, please no. Don't declare an `err` var at the top level describe or at any scope that gets shared.

You didn't do this but since there will be changes, please address this as well.